platform/security/filter/security-filter-login/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -16,7 +16,8 @@
 <metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
 
     <OCD name="Session" id="org.codice.ddf.security.filter.login.Session">
-        <AD description="The number of minutes after a session has been inactive that it should be invalidated."
+        <AD description="Specifies the length of inactivity (in minutes) between client requests before the servlet container will invalidate the session (this applies to all client sessions).
+            This value must be 2 minutes or greater, as users are warned when only 1 minute remains. If a value of less than 2 minutes is used, the timeout is set to the default time of 15 minutes."
```

Comment:
✏️ Should the default here be 31? Same with the .adoc file

Reply:
Yeah. I missed that.